kubernetes/minikube/install-tools-linux.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+HELM_VERSION="v3.5.3"
+ISTIO_VERSION="1.8.4"
+
+function myinstall()
+{
+    chmod 755 "$1"
+    case "$PATH" in
+        *"$HOME"/.local/bin*)
+            mv -f "$1" "$HOME/.local/bin/$1"
+            ;;
+        *"$HOME"/bin*)
+            mv -f "$1" "$HOME/bin/$1"
+            ;;
+        *)
+            sudo chown root:root "$1"
+            sudo mv "$1" "/usr/local/bin/$1"
+            ;;
+    esac
+}
+
+echo
+echo
+echo "*** Installing VirtualBox ***"
+if [ -e /etc/debian_version ]; then
+    sudo apt-get -y update
+    sudo apt-get -y install virtualbox virtualbox-ext-pack pwgen
```

Comment:
VirtualBox :-( let me see if Minikube will start on KVM.

Reply:
I chose VirtualBox because it's available on Linux, Mac and Windows. We can have different VM engines for different host OS, it's just a bit more work to manage them.

Reply:
People like me, who really care enough to ensure to use kvm, can do it easily enough, like I did.  We don't have to worry about official kvm support.